app/plugin.go

```diff
@@ -18,6 +18,10 @@ import (
 	"github.com/pkg/errors"
 )
 
+// PublicKeyFileExtention is the extention of the public key file. Only this file
+// extention is permitted.
+const PublicKeyFileExtention = ".asc"
```

Comment:
Since this is global to the package, propose:
```suggestion
const PluginSignaturePublicKeyFileExtention = ".asc"
```